common/coin_support/coin_utils.h

```diff
@@ -352,6 +351,21 @@ void derive_hdnode_from_path(const uint32_t *path,
                              const uint8_t *seed,
                              HDNode *hdnode);
 
+/**
+ * @brief Generates the node from provided start node at the requested path
+ * @details The function performs hardened derivation of the nodes at the
+ * specified derivation path.
+ *
+ * @param path
+ * @param path_length
+ * @param s_node
+ * @param hdnode
+ */
+bool derive_hdnode_from_node(const uint32_t *path,
+                             size_t path_length,
+                             const HDNode *s_node,
+                             HDNode *hdnode);
+
```

Comment:
Would this function be required as a global function or can it be a static one?

Reply:
this will be removed altogether

Reply:
removed